config/permissions.go

```diff
@@ -0,0 +1,214 @@
+package config
+
+import "gopkg.in/macaroon-bakery.v2/bakery"
+
+var (
+	PricePermissions = []bakery.Op{
+		{
+			Entity: "market",
+			Action: "updatemarketprice",
+		},
+	}
+	MarketPermissions = []bakery.Op{
+		{
+			Entity: "market",
+			Action: "openmarket",
+		},
+		{
+			Entity: "market",
+			Action: "closemarket",
+		},
+		{
+			Entity: "market",
+			Action: "updatemarketstrategy",
+		},
+	}
+	ReadonlyPermissions = []bakery.Op{
+		{
+			Entity: "operator",
+			Action: "reportmarketfee",
+		},
+		{
+			Entity: "operator",
+			Action: "listswaps",
+		},
+		{
+			Entity: "operator",
+			Action: "listmarket",
+		},
+		{
+			Entity: "operator",
+			Action: "balancefeeaccount",
+		},
+	}
+
+	AdminPermissions = []bakery.Op{
+		{
+			Entity: "wallet",
+			Action: "genseed",
+		},
+		{
+			Entity: "wallet",
+			Action: "initwallet",
+		},
+		{
+			Entity: "wallet",
+			Action: "unlockwallet",
+		},
+		{
+			Entity: "wallet",
+			Action: "changepassword",
+		},
+		{
+			Entity: "wallet",
+			Action: "walletaddress",
+		},
+		{
+			Entity: "wallet",
+			Action: "walletbalance",
+		},
+		{
+			Entity: "wallet",
+			Action: "sendtomany",
+		},
+		{
+			Entity: "operator",
+			Action: "depositmarket",
+		},
+		{
+			Entity: "operator",
+			Action: "listdepositmarket",
+		},
+		{
+			Entity: "operator",
+			Action: "depositfeeaccount",
+		},
+		{
+			Entity: "operator",
+			Action: "balancefeeaccount",
+		},
+		{
+			Entity: "operator",
+			Action: "openmarket",
+		},
+		{
+			Entity: "operator",
+			Action: "closemarket",
+		},
+		{
+			Entity: "operator",
+			Action: "listmarket",
+		},
+		{
+			Entity: "operator",
+			Action: "updatemarketfee",
+		},
+		{
+			Entity: "operator",
+			Action: "updatemarketprice",
+		},
+		{
+			Entity: "operator",
+			Action: "updatemarketstrategy",
+		},
+		{
+			Entity: "operator",
+			Action: "withdrawmarket",
+		},
+		{
+			Entity: "operator",
+			Action: "listswaps",
+		},
+		{
+			Entity: "operator",
+			Action: "reportmarketfee",
+		},
+	}
+)
+
+// RPCServerPermissions returns a mapping of the main RPC server calls to
+// the permissions they require.
+func RPCServerPermissions() map[string][]bakery.Op {
```

Comment:
shouldn't it be?
```suggestion
func RPCServerPermissions() map[string]bakery.Op {
```

I didn't get why mapping each RPC path to a slice of permissions.